test/argsparser_test.ts

```diff
@@ -108,15 +108,15 @@ Deno.test({
 const commandArgs2: Args[] = [
   {
     name: 'user',
-    match: 'mentionUser'
+    match: 'user'
   },
   {
     name: 'channel',
-    match: 'mentionChannel'
+    match: 'channel'
   },
   {
     name: 'role',
-    match: 'mentionRole'
+    match: 'channel'
```

Comment:
Mention role to channel?

Reply:
fixed, i think i messed up when using find and replace